Lib/multiprocessing/reduction.py

```diff
@@ -51,14 +51,35 @@ def dumps(cls, obj, protocol=None):
         cls(buf, protocol).dump(obj)
         return buf.getbuffer()
 
-    loads = pickle.loads
+    @classmethod
+    def loads(cls, bytes_object, *, fix_imports=True,
+                   encoding="ASCII", errors="strict"):
+        return pickle.loads(bytes_object, fix_imports=fix_imports,
+                            encoding=encoding, errors=errors)
+
+
+class AbstractPickler(ForkingPickler):
+    pass
+
+
+def register(type_, reduce_):
+    return ForkingPickler.register(type_, reduce_)
 
-register = ForkingPickler.register
 
 def dump(obj, file, protocol=None):
     '''Replacement for pickle.dump() using ForkingPickler.'''
     ForkingPickler(file, protocol).dump(obj)
 
+
+def dumps(obj, protocol=None):
+    return ForkingPickler.dumps(obj, protocol=protocol)
+
+
+def loads(s, *, fix_imports=True, encoding="ASCII", errors="strict"):
+    return ForkingPickler.loads(s, fix_imports=fix_imports,
```

Comment:
By the way, I see that `sharedctypes` is still using `_ForkingPickler` directly. Should it be fixed as well?